spec/policies/report_policy_spec.rb

```diff
@@ -1118,6 +1238,22 @@
       end
     end
 
+    it_behaves_like("when current user is a DDFIP form admin for creation_local_professionnel") do
+      it "scopes on reports transmitted to the DDFIP with type creation_local_professionnel" do
+        expect {
+          scope.load
+        }.to perform_sql_query(<<~SQL)
+          SELECT     "reports".*
+          FROM       "reports"
+          WHERE "reports"."discarded_at" IS NULL
+            AND "reports"."sandbox" = FALSE
+            AND "reports"."state" IN ('transmitted', 'acknowledged', 'accepted', 'assigned', 'applicable', 'inapplicable', 'approved', 'canceled', 'rejected')
+            AND "reports"."ddfip_id" = '#{current_organization.id}'
+            AND "reports"."form_type" = 'creation_local_professionnel'
+        SQL
+      end
+    end
+
```

Comment:
Ajouter les tests : 
* "when current user is a DDFIP admin with form types assigned"
* "when current user is a DDFIP form admin and a Super Admin"